front/pages/api/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -269,10 +269,19 @@ async function handler(
         // If we have a `wIdTarget` query parameter, we are fetching runs that were created with an
         // API key coming from another workspace. So we override the `owner` variable and check that
         // the user is a user of that workspace.
-        const target = await Authenticator.fromSession(
+
+        // Dust super users can view runs of any workspace.
+        let target = await Authenticator.fromSuperUserSession(
           session,
           req.query.wIdTarget as string
         );
+        if (!target.isUser()) {
```

Comment:
let's do isAdmin here?

Reply:
Changed (it doesn't really change anything though as its either admin or none)